From c8df6599c6bdb2c8b84fe5e988acf14ac6ea3aa1 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Thu, 14 Jul 2022 06:25:39 +0000
Subject: libstdc++: fix incomplete type of media::ScreenEnumerator

Destructor of std::unique_ptr in libstdc++ uses sizeof() which
requires full definition of media::ScreenEnumerator for return type
of content::ContentBrowserClient::CreateScreenEnumerator().

Bug: 957519
Change-Id: I11379cf07b32436e532a4a1be8b3d669f30d3b8f
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3757663
Reviewed-by: John Abd-El-Malek <jam@chromium.org>
Commit-Queue: Stephan Hartmann <stha09@googlemail.com>
Cr-Commit-Position: refs/heads/main@{#1024132}
---
 content/public/browser/content_browser_client.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/content/public/browser/content_browser_client.cc b/content/public/browser/content_browser_client.cc
index 8654b944e2d9c..e58cb63afeeec 100644
--- a/content/public/browser/content_browser_client.cc
+++ b/content/public/browser/content_browser_client.cc
@@ -50,6 +50,7 @@
 #include "content/public/common/content_features.h"
 #include "content/public/common/url_utils.h"
 #include "media/audio/audio_manager.h"
+#include "media/capture/content/screen_enumerator.h"
 #include "media/mojo/mojom/media_service.mojom.h"
 #include "net/cookies/site_for_cookies.h"
 #include "net/ssl/client_cert_identity.h"
-- 
2.34.1

